tests/test_init_command.rs

```diff
@@ -112,6 +112,18 @@ fn test_init_git_external() {
     "###);
 }
 
+#[test]
+fn test_init_git_external_bad_path() {
+    let test_env = TestEnvironment::default();
+    let stderr = test_env.jj_cmd_failure(
+        test_env.env_root(),
+        &["init", "repo", "--git-repo", "non-existent"],
+    );
+    insta::assert_snapshot!(stderr.replace('\\', "/"), @r###"
```

Comment:
`.replace('\\', "/")` looks redundant since the whole path should be captured by normalization pattern.

Reply:
Yes, good catch. That's a leftover from an earlier version where I only replaced `$ENV_ROOT` itself. Then I figured it's better to include the path after it too (because I saw that tests failed on Windows :))